src/pages/GE_TokenizeCard.page

```diff
@@ -87,10 +88,15 @@
         * @description Creates a token and posts the response to a credit card widget LWC
         */
         function handleTokenCreation() {
-// TODO Remove this line in W-8021204
-console.log('>>> Tokenize with Name On Card: "' + nameOnCard + '"');
+            // TODO Remove this line in W-8021204
+            console.log('>>> Tokenize with Name On Card: "' + nameOnCard + '"');
```

Comment:
@RadSFDeveloper @dfuller114 - do we want this console log to stay in here? Do we have a plan to keep it from hitting production?

Reply:
Is there any issue in keeping the debug log as is? There is a future work on the RD entry form to provide card holder name to the tokenization, so it is beneficial to keep it for that purpose. However, if there is any issue with the debug statement being included in the release, feel free to remove it in this PR.
CC @force2b

Reply:
We generally don't ship code with log statements, but if it's serving a purpose I'm ok with it remaining on a temporary basis!

Reply:
Since it is a temp debug and it can be added when the future work starts, we can add the console output just for that WI, so @dfuller114 , do you mind removing the debug statement if it is not too late?

Reply:
Agreed. We can just add it back when needed for QA.

Reply:
just pushed an update to remove the console log